commit f4a175a625e3e98b24928aeac249ce7aabe64dc5
Author:     Sebastian Neuser <haggl@sineband.de>
AuthorDate: Mon Jan 15 02:41:06 2018 +0100
Commit:     Sebastian Neuser <haggl@sineband.de>
CommitDate: Mon Jan 15 02:44:53 2018 +0100

    Fix connect-to-mainnet generator
    
    A typo in the check for existence of the TLS certificate directory
    causes the server cert to re-generated every time. This patch fixes
    the situation.

diff --git a/scripts/launch/connect-to-cluster/default.nix b/scripts/launch/connect-to-cluster/default.nix
index 3e76280ba..c66fb70d2 100755
--- a/scripts/launch/connect-to-cluster/default.nix
+++ b/scripts/launch/connect-to-cluster/default.nix
@@ -63,7 +63,7 @@ in pkgs.writeScript "${executable}-connect-to-${environment}" ''
 
   echo "Launching a node connected to '${environment}' ..."
   ${ifWallet ''
-  if [ ! -d ${stateDir}tls ]; then
+  if [ ! -d ${stateDir}/tls ]; then
     mkdir ${stateDir}/tls/
     openssl req -x509 -newkey rsa:2048 -keyout ${stateDir}/tls/server.key -out ${stateDir}/tls/server.cert -days 3650 -nodes -subj "/CN=localhost"
   fi
